app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -3,11 +3,11 @@
 
   <% if follow_request %>
     <% if follow_request.pending? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
+      <%= link_to follow_request,  remote: true, method: :delete, class: "btn btn-outline-secondary #{dom_id(follow_request)}" do %>
```

Comment:
This took me a whole day to figure it out to target class instead of id in this specific project.